### DISCUSSION TITLE
Issue after Ventura Install Black screen

### DISCUSSION THREAD

**Discussion by sohail579** [osy86/HaC-Mini#1050] [Q&A]
I have Installed Ventura on my NUC, I have 2 NVME Drives one of them has Windows 11 installed on it.  I then installed Ventura on the other and it all worked fine I installed the HacMini.pkg and enabled boot securely as per the blog instructions.  I then restarted and now I just get a black screen as soon as the NUC turns on and nothing else and no responses.  I feel as though its not even posting

**Comment by osy** [comment]
Did you follow these steps https://osy.gitbook.io/hac-mini-guide/post-installation/support#black-screen-powering-up-with-tb-dp-display-or-tb-port-not-working ?

**Reply by sohail579** [reply]
I did yes but i have now had to return it and get a warranty replacement not sure what happened i think there was another hardware issue which happened and its was a coincidence
